python/graphstorm/gconstruct/remap_result.py

```diff
@@ -49,6 +49,87 @@
 # id_maps to each worker process.
 id_maps = {}
 
+def worker_remap_node_emb(emb_file_path, nid_path, nid_range_info, ntype,
+    output_fname_prefix, chunk_size, preserve_input):
+    """ Do one node prediction remapping task
+
+        GraphStorm has two ways to store node embeddings:
+        1. Store the entire node embeddings of nodes of ntype T.
+           In this case, node embeddings has already been shuffled
+           into the order before doing graph partition.
+           The order is aligned with the order in the
+           original-to-graphstorm ID mapping.
```

Comment:
what does it mean?

Reply:
The order of nodes has already been shuffled according to partitioned node ID to global node ID mapping.